requirements.txt

```diff
@@ -0,0 +1,10 @@
+boto3>=1.10.49
```

Comment:
why are we adding these here instead of setup.py?  Are they deploy-only requirements?

Reply:
I use it install python deps locally so pylint can pick them up.  It should be in the package anyway, but it is not used by app install.